key.ui/src/main/java/de/uka/ilkd/key/gui/extension/impl/KeYGuiExtensionFacade.java

```diff
@@ -221,6 +223,7 @@ public static List<KeYGuiExtension.Toolbar> getToolbarExtensions() {
      */
     public static List<JToolBar> createToolbars(MainWindow mainWindow) {
         return getToolbarExtensions().stream().map(it -> it.getToolbar(mainWindow))
+                .peek(x -> x.setFloatable(false))
```

Comment:
I am ok with setting all toolbars to non-floatable, as I have never used the possibility to rearrange them. I think it is indeed a good idea to remove this. However, since this makes the layout more restrictive than earlier, we should ask others for feedback.
@KeYProject/reviewers What do you think? Has anyone ever used this feature productively?

Reply:
This a feature, that is not very popular amongst Swing applications. I have no obstacles, and we can revert this easily.